RocketChat/Rocket.Chat
# [FIX] Full name not applied even if `useRealName` is true after reacting

Names are not stored along with the reactions when `setReaction` method is called. This is way usernames were displayed in the Reactions even when `useRealName` is set true. See attached video for more info.


https://user-images.githubusercontent.com/77742477/188212000-b81ffa32-5dca-4ea9-95ac-568f86a92536.mp4

https://user-images.githubusercontent.com/804994/187958874-470ec04e-a409-4b68-97f6-d2370b9e5329.mp4






## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce


* Make sure `useRealName` is turned on for the server.
* See reactions on a message. It'll show full names of the users.
* Now react to that message and open reactions again. It'll show usernames instead of Full Names.

## Further comments
